main.cc

```diff
@@ -154,6 +152,13 @@ int main(int argc, char** argv) {
         } else if (ev.type == SDL_MOUSEBUTTONDOWN &&
                    ev.button.button == SDL_BUTTON_LEFT) {
           dragged = sys.ObjectFor(ev.button.x, ev.button.y);
+	  if(dragged->isToolbar())
+	    continue;
```

Comment:
This skips the part of the main loop that does stuff other than process events (i.e. update screen render, move neutrons, etc.) - need to use alternate way to skip without omitting code outside event processing if-elseif.